### PR TITLE
Experience Composer

### PR DESCRIPTION
**Pull request by behei-vonage**
This PR adds sample requests for the OpenTok Experience composer: 

* it shows how to start a composer
* it shows how to get information about a particular composer
* it shows how to get a list of experience composers
* It shows how to stop a composer

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -279,4 +279,121 @@ router.get('/archive', function (req, res) {
   });
 });
 
+/**
+ * POST /render
+ */
```
Comment (manchuck) [inline routes/index.js:284]: This comment was brought to you by the Department of Redundancy Department

File: routes/index.js [hunk 2]
```diff
@@ -279,4 +279,121 @@ router.get('/archive', function (req, res) {
...
+    console.warn(err);
+    res.status(500);
+    res.send(`Error starting Experience Composer: ${err}`);
+    return;
+  }
+});
+
+/**
+ * GET /render/info
+ */
```
Comment (manchuck) [inline routes/index.js:319]: This comment is redundant

File: routes/index.js [hunk 3]
```diff
@@ -279,4 +279,121 @@ router.get('/archive', function (req, res) {
...
+    console.warn(err);
+    res.status(err.status);
+    res.send(`Error retrieving composer information: ${err}`);
+    return;
+  }
+});
+
+/**
+ * GET /render/list
+ */
```
Comment (manchuck) [inline routes/index.js:347]: Another redundant comment

**Review by manchuck — CHANGES_REQUESTED** [review]
These comments are not blockers for the PR:

- I would also start using fat arrow functions to help improve GC
- Going forward, we are looking to use `node-fetch` instead of `axios`. This way it's more in sync with the browser API

File: routes/index.js (hunk 1 above)
Comment (behei-vonage) [reply routes/index.js:284]: I just followed the convention that was already in this repo. I'll remove it but other routes need to be updated as well

**Review by manchuck — APPROVED** [review]

**Review by michaelwhittemore — APPROVED** [review]

**Pull request merged**